### DISCUSSION TITLE
2024-08-12 회의록

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#251] [회의록]
### 🔥 추가 논의가 필요한 사항(리마인드)
- **알림 아이콘 위치 - 계속 생각해보기**
- **글쓰기 아이콘 위치 - 계속 생각해보기(하단바중앙으로?)**
- **pwa 더 고민해보기**
- 디자이너님 피그마에 px 단위로 적어주신다고 함

### 🔥 티켓 재분배(리마인드)
- 일정 체크 및 티켓 재분배 완료
- 8/15 목표로 달리기~
- 8/12 월요일 코드 리뷰 및 UT 당첨자 선정

### 🔥 MVP/UT 후 2주간 이어서 해결해야 할 내용들(리마인드)
- 남은 기능들(찜하기, 좋아요, 팔로우, 알림)
  - 스토리 / 찜하기 / 좋아요 / 글쓰기 수정- 오은 
  - 리치텍스트에디터 / 글쓰기 삭제 / 팔로우 / 온도지수 로직 - 김병준
  - 알림(팔로우찜하기되는대로) - 김민지
  - 검색 - 곽현정
- 리팩토링 / 컨벤션
  - 탠스택쿼리로 전환(인밸리데이트쿼리)
- 디렉토리(컴포넌트) 구조 정리
- 데스크탑 버전

## 🍀 공통 트러블 슈팅

// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>

**Comment by Captain-Kim** [comment]
**슬랙 전체잡담당 UT 글** 전체 취소선, 마강 안내요

**Comment by Captain-Kim** [comment]
**상품수령대상자 선정 투표**

## UT 상품 수령자 동점자 처리 방법

배민 상품권 1~3등 // 3, 4, 6, 14, 15 중 재투표해야 하나, 카페라떼도 동점자가 많음.
1) 동점자 전체 재투표
2) 1, 2, 3 등만 재투표 후 카페라떼는 접수순 또는 사다리
3) 전체 접수순
3) 전체 사다리

**Comment by Captain-Kim** [comment]
## 김병준 추가 내용

### 브로셔에 포함된 UT 결과보고서는 누가 만드는가?

=> 디자이너님이 UT 내용을 모두 이해하기 어려우실 것 같으니, 본인이 초안이라도 제작하겠음.
=> ++) 다시 보니 필수는 아닌듯 하나 이왕 분석해놓은 것 활용해서 제작하겠음.
